uopz.c

```diff
@@ -24,7 +24,11 @@
 #include "php_ini.h"
 #include "ext/standard/info.h"
 
-#include "SAPI.h"
+#ifndef HAVE_SPL
+/* {{{ */
+zend_class_entry *spl_ce_RuntimeException;
+zend_class_entry *spl_ce_InvalidArgumentException; /* }}} */
+#endif
```

Comment:
Please don't readd this SPL thing, not needed

Reply:
Fixed (did I mention I had no idea about the C code)